app/src/main/java/org/schabi/newpipe/local/holder/LocalStatisticStreamItemHolder.java

```diff
@@ -143,13 +148,19 @@ public void updateState(final LocalItem localItem,
 
         if (item.getProgressMillis() > 0 && item.getStreamEntity().getDuration() > 0) {
             itemProgressView.setMax((int) item.getStreamEntity().getDuration());
-            if (itemProgressView.getVisibility() == View.VISIBLE) {
-                itemProgressView.setProgressAnimated((int) TimeUnit.MILLISECONDS
-                        .toSeconds(item.getProgressMillis()));
+            if (DependentPreferenceHelper
+                    .getPositionsInListsEnabled(itemProgressView.getContext())) {
+                if (itemProgressView.getVisibility() == View.VISIBLE) {
+                    itemProgressView.setProgressAnimated((int) TimeUnit.MILLISECONDS
+                            .toSeconds(item.getProgressMillis()));
+                } else {
+                    itemProgressView.setProgress((int) TimeUnit.MILLISECONDS
+                            .toSeconds(item.getProgressMillis()));
+                    ViewUtils.animate(itemProgressView, true, 500);
+                }
             } else {
                 itemProgressView.setProgress((int) TimeUnit.MILLISECONDS
                         .toSeconds(item.getProgressMillis()));
```

Comment:
Same as above. I don't get why this is here, and the `getPositionsInListsEnabled` condition should be moved to the `if` on line 149